build/common.props

```diff
@@ -17,9 +17,6 @@
     <RepositoryUrl>https://github.com/Azure/azure-webjobs-sdk</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
-    <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
```

Comment:
Do we need this change?

Reply:
This is for my local setup to stay happy - I will revert changes to csproj and props etc. later